contrib/drivers/lvm/targets/iscsi.go

```diff
@@ -86,8 +86,9 @@ func (t *tgtTarget) getTgtConfPath(volId string) string {
 type configMap map[string][]string
 
 func (t *tgtTarget) CreateISCSITarget(volId, tgtIqn, path, hostIp, initiator string, chapAuth []string) error {
-	if hostIp == "" {
-		return errors.New("create ISCSI target failed, host ip cannot be empty")
+	// Multi-attach require a specific ip
+	if hostIp == "" || hostIp == "ALL" {
+		return errors.New("create ISCSI target failed: host ip cannot be empty")
```

Comment:
the error is not corrct.

Reply:
Besides, some log msg may be required here.

Reply:
done.